manifests/j/JPSoft/tcmd/32.10.21.0/JPSoft.tcmd.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=QUSU.CRLF.7-5-0-preview-5.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: JPSoft.tcmd
+PackageVersion: 32.10.21.0
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /exenoui /qn
+  SilentWithProgress: /exebasicui /qb
+UpgradeBehavior: install
+ProductCode: '{B65188D1-31F3-4BB9-85E8-A671B7A43F4F}'
+Installers:
+- Architecture: x64
+  # InstallerUrl: https://jpsoft.com/downloads/v32/tcmd.exe
+  InstallerUrl: http://asus-gt302/d/downloads/tcmd.exe
```

```suggestion
  InstallerUrl: https://jpsoft.com/downloads/v32/tcmd.exe
```

Silly me 🤣 Forget to switch back to official URL.